client/components/PhotoCard.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PhotoCard = ({ url, index, clickPhotoOpen }) => (
+  <div className="photo-card" onClick={clickPhotoOpen}>
+    <img src={url} data-index={index} alt="this is a description" />
```

Comment:
Love the placeholder "this is a description" there

Reply:
yeah, ESLint made me do it :)